contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,212 @@
+    - [Fill out the issue](#fill-out-the-issue)
+    - [Iterate](#iterate)
+
+
+## Monitoring the health of Kubernetes with TestGrid
+
+TestGrid is a highly-configurable, interactive dashboard for viewing your test
+results in a grid, see https://github.com/GoogleCloudPlatform/testgrid.
+
+The Kubernetes community has its own instance of TestGrid, https://testgrid.k8s.io/,
+which we use to monitor and observe the health of the project.
+
+Each SIG has its own set of dashboards and each dashboard is composed of
+different end-to-end (e2e) jobs.
+E2E jobs are in turn made up of test stages (i.e., bootstraping a Kubernetes
+cluster, tearing down a Kubernetes cluster) and e2e tests (i.e., Kubectl client
+Kubectl logs should be able to retrieve and filter logs).
+These views allow all different teams to monitor and understand how their areas
+are doing.
+
+We highly enourage any one to take periodically monitor these dashboards.
+If you see that a job or test has been failing please raise an issue with the
+corresponding SIG in either their mailing list of in slack.
+
+Help maintaining tests, fixing broken tests, improving test success rates, and
+overall test improvements are always highly needed throughout the project.
+
+**Note**: It is important that all SIGs periodically monitor their jobs and
+tests. These are used to figure out when to release Kubernetes.
+Furthermore, if jobs or tests are failing or flaking, then pull requests will
+take a lot longer to be merged.
+
+
+### What dashboards should I monitor?
+
+This depends on what areas of Kubernetes you want to contribute to.
+You should monitor the dashboards owned by the SIG you are working with.
+Additionally, you should check
+
+* https://testgrid.k8s.io/sig-release-master-blocking and
+* https://testgrid.k8s.io/sig-release-master-informing
+
+Since these jobs run tests owned by other SIGs.
+Also make sure to periodically check on the "blocking" and "informing"
+dashboards for past releases.
+
+---
+
+## What do I do when I see a TestGrid alert?
+
+If you are part of a SIG's mailing list, occasionally you may see email from
+TestGrid reporting that a job or a test has recently failed.
+If you are casually browsing through TestGrid, you may also see jobs labeled as
+"flaky" (in purple) or as "failing" (in red).
+This section is to help guide you on what to do in these occasions.
+
+### Communicate your findings
+
+The number one thing to do is to communicate your finding (a test or job has
+been flaking or failing).
+If you saw a TestGrid alert on a mailing list, please reply to the thread and
+mentioned that you are looking into it.
+It is important to communicate to prevent duplicate work and to ensure CI
+problems get attention.
+
+In order to communicate with the rest of the community and to drive the work,
+please open up an issue on Kubernetes,
+https://github.com/kubernetes/kubernetes/issues/new/choose , chose the appropriate issue
+template.
+
+### Fill out the issue
```

Comment:
Unsure if this is out of scope for this issue or whether it falls more under "revamping the issue template"? -> We should talk a little bit about how to title it, e.g. prefix with `[Failing Test]` or `[Flaky Test]` (depending on what's happening)

Reply:
I said something similar in a different PR review that wrote instructions on how to file a flake issue https://github.com/kubernetes/community/pull/5205#discussion_r502665584

Instructions on correctly filling out an issue are most likely to be read if they are part of the issue template itself.  Alternatively, make a page dedicated just to how to file kubernetes/kubernetes issues, and link to that page from the issue template.

I opened https://github.com/kubernetes/kubernetes/issues/95528 to cover updating the flake template, maybe it should expand for both.